include/openenclave/enclave.h

```diff
@@ -579,6 +580,100 @@ oe_result_t oe_get_seal_key_v2(
  */
 void oe_free_seal_key(uint8_t* key_buffer, uint8_t* key_info);
 
+/**
+ * Get enclave-specific key information that could be passed to
+ * oe_get_seal_key_v2() to derive a symmetric encryption key coupled to the
+ * enclave platform.
+ *
+ * @param[in] seal_policy The policy for the identity properties used to derive
+ * the seal key.
+ * @param[in] entropy Optional parameter containing additional entropy for key
+ * derivation. If \c NULL, oe_get_seal_key_info() will generate entropy on
+ * behalf of the caller.
+ * @param[in] entropy_size Must be \c 0 if \p entropy is \c NULL.
+ * @param[in] opt_flags \c 0 to use TEE defaults. If \c OE_SEAL_SGX_SPECIFIC is
+ * specified, the rest of \p opt_flags is interpreted as the \a attribute_mask
+ * for \c EGETKEY on SGX.
+ * @param[out] key_info On success this points to the enclave-specific key
+ * information which can be used to retrieve the key by passing it to
+ * oe_get_seal_key_v2(). Freed by calling oe_free_key().
+ * @param[out] key_info_size On success, this is the size of the \p key_info
+ * buffer.
+ *
+ * @retval OE_OK The seal key info was successfully requested.
+ * @retval OE_INVALID_PARAMETER At least one parameter is invalid.
+ * @retval OE_OUT_OF_MEMORY Failed to allocate memory.
+ */
+oe_result_t oe_get_seal_key_info(
+    oe_seal_policy_t seal_policy,
+    uint8_t* entropy,
+    size_t entropy_size,
+    uint64_t opt_flags,
+    uint8_t** key_info,
+    size_t* key_info_size);
+
+/**
+ * Seal data to an enclave using AEAD (Authenticated Encryption with Additional
+ * Data).
+ *
+ * @param[in] key_info Key info to be passed to oe_get_seal_key_v2() to derive
```

Comment:
This seems like an odd description since it's being passed into oe_seal not oe_get_seal_key_v2, and "to be passed" would imply the caller has to do so, which seems confusing here.

Reply:
Will update the comments.